Refit/RequestBuilderImplementation.cs

```diff
@@ -138,7 +138,19 @@ public RequestBuilderImplementation(Type targetInterface, RefitSettings refitSet
                     // for anything that fell through to here, if this is not
                     // a multipart method, add the parameter to the query string
                     if (!restMethod.IsMultipart) {
-                        queryParamsToAdd[restMethod.QueryParameterMap[i]] = settings.UrlParameterFormatter.Format(paramList[i], restMethod.ParameterInfoMap[i]);
+                        var attr = restMethod.ParameterInfoMap[i].GetCustomAttribute<QueryAttribute>() ?? new QueryAttribute();
+                        if (DoNotConvertToQueryMap(paramList[i]))
+                        {
```

Comment:
:arrow_up: 

The style the project uses can be a little hard to follow, but the gist is that we don't use access modifiers where they're unnecessary (e.g. `private`) and we use the [K&R indent style](https://en.wikipedia.org/wiki/Indent_style#K.26R) i.e. opening bracket on the same line as the condition.